backend/src/RedisDriver.js

```diff
@@ -0,0 +1,29 @@
+const Redis = require( 'ioredis' );
+const redisAdapter = require( 'socket.io-redis' );
+const dotenv = require( 'dotenv' );
+dotenv.config( { path: '.env' } );
+
+class RedisDriver {
+	constructor( io ) {
+		this.io = io;
+		this._config = {
+			host: process.env.REDIS_HOST,
+			port: process.env.REDIS_PORT
+		};
+		this._client = this._connect( io );
+	}
+
+	get client() {
+		return this._client;
+	}
+
+	_connect( io ) {
+		const client = new Redis( this._config );
+		io.adapter( redisAdapter( this._config ) );
```

Comment:
Hmmm, also I'm not sure that it should be here because now this class/method has two responsibilities.

I would prefer to have something similarly:

```
const io = io;
const redisDriver = new RedisDriver();

await redisDriver.connect();

io.adapter(redisDriver.client)
```

☝️ this is the only idea ;)

Reply:
Done, you can see it in `c8d99f6`.